test/test_other.py

```diff
@@ -3568,6 +3568,9 @@ def test_js_lib_no_override(self):
     err = self.expect_fail([EMCC, 'duplicated_func.c'] + self.get_emcc_args())
     self.assertContained('error: Symbol re-definition in JavaScript library: duplicatedFunc. Do not use noOverride if this is intended', err)
 
+  def test_override_stub(self):
+    self.do_run_from_file(test_file('other/test_override_stub.c'), test_file('other/test_override_stub.out'))
```

Comment:
I think you can just do `self.do_other_test('yest_override_stub.c')` here since your test falls into this common pattern